content/ui-patterns/feature-onboarding.mdx

```diff
@@ -0,0 +1,271 @@
+---
+title: Feature onboarding
+---
+
+import {Box, Flex, BorderBox, Text} from '@primer/components'
+
+This is a guide for designing onboarding for the product and does not include what to do for marketing pages, email announcements, social media, etc.
+
+Onboarding is a virtual unboxing experience that helps users get started with an app.
+
+- ✅ Think creatively utilizing different ways to onboard users, such as empty states and off-product avenues.
+- ✅ For feature annoucements, prepare a release timeline to call out new features.
+- 🚫 Do not leave feature annoucement campaigns running forever.
+
+# Guiding principles
+## Proximity and proportion
+Onboarding elements should maintain an appropriate proximity to where the feature will live in perpetuity.
+
+Consider the primary job to be done on a page before disrupting the user with a feature callout.  Keep in mind the context, placement, and dominance of an onboarding element. Onboarding UI choices should be proportional to the hierarchy of the feature on the page.
+
+## Set clear campaign limits
+Use feature onboarding UI for a limited time, make it clear to the user how they may dismiss the message and return to their original task quickly. Define clear paramaters for why, when, and how often a user may see a onboarding message:
+
+- **Define triggers**
+  - System tiggers, e.g. a new feature released, a user should not need to dismiss the same annoucement each time they visit a different repository.
+  - User triggers, e.g. a user visits a specific repository for the first time, if recurring, a user should feel added value each time they see the message.    
+- **Timebox**
+  - Maximum number of days a coordinated campaign will run across all targeted users.
+  - Maximum number of times a user will see an in-product message until dismissed.
+
+## Avoid traffic collisions
+Avoid showing too many alerts at the same time. Collisions are awkward and undesireable. Consider what other alerts, teaching bubbles, or banners may be active on the page before launching your onboarding experience.
```

Comment:
Another list, ie:
- Avoid showing too many alerts at the same time, because collisions are awkward and undesirable [typo here btw, is there a max number to be more specific?]
- Consider what other alerts, teaching...